pilot/cmd/pilot-agent/status/server_test.go

```diff
@@ -909,6 +909,10 @@ func TestAppProbe(t *testing.T) {
 				}
 			}
 		}
+		if host := req.Header.Get("Host"); host != "" {
```

Comment:
nit: if is redundant. just assign.

Reply:
If you mean req.Host = req.Header.Get("Host"), then we couldn't we set an empty Host?

Reply:
Sorry, my bad. I didn't think Host can have a value from before.